### DISCUSSION TITLE
Get current URL dynamically

### DISCUSSION THREAD

**Discussion by naptestdev** [sveltejs/kit#2830] [Q&A] [answered]
### Describe the problem

I couldn't get the current URL in the sveltekit app

### Describe the proposed solution

Create a new hook or something that can support this

### Importance

i cannot use SvelteKit without it

**Comment by ignatiusmb** [comment, marked as answer]
Use `page` stores from [`$app/stores`](https://kit.svelte.dev/docs#modules-$app-stores)
